libraries/wutdevoptab/devoptab_fsa_fchmod.cpp

```diff
@@ -0,0 +1,10 @@
+#include "devoptab_fsa.h"
+
+int
+__wut_fsa_fchmod(struct _reent *r,
```

Comment:
Since this function only returns ENOSYS, it can be removed. Default devoptab behavior for unimplemented functions is to just return ENOSYS.